doc/gcylcrc.tex

```diff
@@ -65,6 +65,18 @@ \subsubsection{initial side-by-side views}
 \item {\em default:} ``False''
 \end{myitemize}
 
+\subsubsection{task filter exclude}
+
+Set the initial filtering options when the GUI starts. This can later be
+changed by using the "View" menu "Task Filtering" option.
```

Comment:
? Maybe needs to be a little more explicit, e.g.:
> Later this can be changed using the "Task Filtering" option under the gcylc "View" menu, or clicking on the filtering indicator in the application status bar.